backend/src/main/java/ai/verta/modeldb/experimentRun/subtypes/PredicatesHandler.java

```diff
@@ -75,6 +76,11 @@ public PredicatesHandler() {
         // case visibility:
       case "":
         return InternalFuture.failedStage(new InvalidArgumentException("Key is empty"));
+      case "end_time":
+        return InternalFuture.completedInternalFuture(
+            new QueryFilterContext()
+                .addCondition("experiment_run.end_time = :" + bindingName)
+                .addBind(q -> q.bind(bindingName, value.getStringValue())));
```

Comment:
filter by `end_time`

Reply:
Only equality?

Reply:
We have only equality check in test cases so if you think there will be have other operator as well then i can handle it.